packages/dialtone-css/lib/build/less/recipes/leftbar_row.less

```diff
@@ -342,7 +342,7 @@
   animation-fill-mode: both;
 }
 
-@keyframes opacity-pulsate {
```

Comment:
Strange this was defined 3 times in a row.. There wasn't a reason for that right?